VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Private/VitruvioModule.cpp

```diff
@@ -491,6 +500,7 @@ FAttributeMapResult VitruvioModule::EvaluateRuleAttributesAsync(const TArray<FIn
 		const RuleFileInfoUPtr StartRuleInfo(prt::createRuleFileInfo(RuleFileUri));
 		const std::wstring StartRule = prtu::detectStartRule(StartRuleInfo);
 
+		FScopeLock Lock(&PrtCacheLock);
```

Comment:
I think it's fine. It's basically just the three last lines which return the values which would not be needed. But adding an extra scope because of that will also require more temporary variables instead so I left it out.

Reply:
Alright, let's keep it this way then.